### DISCUSSION TITLE
Cannot apply to github student developer pack

### DISCUSSION THREAD

**Discussion by Maksewsha** [community/community#38918] [GitHub Education] [answered]
### Select Topic Area

Question

### Body

My university email is valid and my user id is until august 2023 and it is at the screen but i get the error:
Your application cannot be reviewed until you fix the following:
The image you selected does not appear to contain your school name. You may include multiple documents in your image, so long as they are legible. For example, if your original document is not in English then you might take a photo of the original next to an English translation.
You are significantly more likely to be verified if you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name and a short bio.
Have you completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with all your relevant information, such as your full name as it appears in your image and a short bio?

**Comment by samus-aran** [comment, marked as answer]
Hi @Maksewsha 👋 

The rejection reasons are suggesting that the image appears to not contain the name of your school. Compare your evidence with the name of the school you have selected in the application. Do they match? If they do then it may be a case of incorrect translation or poor image quality (ensure you proof is clean, eligible and in focus when taking a photo/upload) . Is your evidence in a non English language? If so then as the suggestion provides the next steps - take another photo with your Student ID next to an English translation.

The other more minor points are still good to resolve if possible.

Let us know how you get on 👍

**Comment by DemonicDev** [comment]
![image](https://user-images.githubusercontent.com/61244099/236649198-64d4981a-610e-4d6f-acf7-e979a2867799.png)
I cant click on Continue, but i need PHPStorm to finish one of my projects 😭, so yeah 😦 i dont know how to re apply for Education pack , since the continue button doesn't work ☹️

**Reply by DemonicDev** [reply]
@samus-aran could u provide help :/ ?
